src/components/Skill.vue

```diff
@@ -94,3 +94,166 @@
 </template>
 
 <script>
+import FrontChart from '../components/FrontChart.vue';
+import BackChart from '../components/BackChart.vue';
+import DevChart from '../components/DevChart.vue';
+export default {
+  components: {
+    FrontChart,
+    BackChart,
+    DevChart
+  },
+  data(){
+    return {
+      frontChange: true,
+      backChange: false,
+      devChange: false
+    }
+  },
+  methods: {
+    F_change(){
+      this.frontChange=!this.frontChange,
```

Comment:
このメソッドの形で行くなら、
```suggestion
      this.frontChange=true;
```
が良いです。front選択中にfront押下するとグラフが消えてしまうので

Reply:
かしこまりました。
修正します！